### PR TITLE
Resolve warnings and fail CI on new warnings

### LINKED ISSUE #77: Warnings when running `hugo server`
```
$ hugo server
Watching for changes in C:\Users\markw\myStuff\dev.luanti.org\{archetypes,assets,content,static,themes}
Watching for config changes in C:\Users\markw\myStuff\dev.luanti.org\hugo.toml
Start building sites …
hugo v0.140.0-3f35721fb2c75a1f7cc5a7a14400b66e73d4b06e+extended windows/amd64 BuildDate=2024-12-17T14:20:55Z VendorInfo=gohugoio

WARN  Raw HTML omitted while rendering "C:/Users/markw/myStuff/dev.luanti.org/content/changelog.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']
WARN  Raw HTML omitted while rendering "C:/Users/markw/myStuff/dev.luanti.org/content/old-changelog.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']
WARN  Raw HTML omitted while rendering "C:/Users/markw/myStuff/dev.luanti.org/content/engine-dev-process/meetings.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']
WARN  Raw HTML omitted while rendering "C:/Users/markw/myStuff/dev.luanti.org/content/translation/weblate.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe    
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']
WARN  Raw HTML omitted while rendering "C:/Users/markw/myStuff/dev.luanti.org/content/translation/mods-and-games.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']

                   | EN   
-------------------+------
  Pages            | 101  
  Paginator pages  |   0  
  Non-page files   |   0  
  Static files     | 135
  Processed images |   0
  Aliases          |  90  
  Cleaned          |   0

Built in 3645 ms
Environment: "development"
Serving pages from disk
Running in Fast Render Mode. For full rebuilds on change: hugo 
server --disableFastRender
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

### PR DESCRIPTION
**Pull request by mark-wiemer**
Closes #77

### CODE REVIEW

File: content/engine-dev-process/meetings.md [hunk 1]
```diff
@@ -1078,7 +1078,7 @@ Arranged by: Krock
 **PR discussion/review**
 
 * Android joystick ([https://github.com/minetest/minetest/pull/7126](https://github.com/minetest/minetest/pull/7126))
-* Formspecs: Add a <use\_color\_alpha> parameter to the box\[\] element ([https://github.com/minetest/minetest/pull/7116](https://github.com/minetest/minetest/pull/7116))
+* Formspecs: Add a `<use\_color\_alpha>` parameter to the box\[\] element ([https://github.com/minetest/minetest/pull/7116](https://github.com/minetest/minetest/pull/7116))
```
Comment (appgurueu) [inline content/engine-dev-process/meetings.md:1081]: I think you should get rid of the `\` escapes here if you want to use inline code: Just `use_texture_alpha`

File: content/engine-dev-process/meetings.md [hunk 2]
```diff
@@ -1108,7 +1108,7 @@ Arranged by: Krock
 * Any new, important bugfixes to backport?
     * Added the \`core.rotate\_node\` bugfix, otherwise probably not. It's pretty much stable.
 * Volunteer for cherry-picking the ToDo backport commits ([https://github.com/minetest/minetest/pull/6746](https://github.com/minetest/minetest/pull/6746))
-    * \`<sfan5> probably me\` ([http://irc.minetest.net/minetest-dev/2018-03-31#i\_5264276](http://irc.minetest.net/minetest-dev/2018-03-31#i_5264276))
+    * \``<sfan5>` probably me\` ([http://irc.minetest.net/minetest-dev/2018-03-31#i\_5264276](http://irc.minetest.net/minetest-dev/2018-03-31#i_5264276))
```
Comment (appgurueu) [inline content/engine-dev-process/meetings.md:1111]: I would just quote or inline code it, no need to preserve the literal backticks imo

File: content/engine-dev-process/meetings.md (hunk 1 above)
Comment (mark-wiemer) [reply content/engine-dev-process/meetings.md:1081]: Yeah, wasn't sure what those were doing in the OG notes either, nbd to me

```suggestion
* Formspecs: Add a `<use_color_alpha>` parameter to the box\[\] element ([https://github.com/minetest/minetest/pull/7116](https://github.com/minetest/minetest/pull/7116))
```

File: content/engine-dev-process/meetings.md (hunk 2 above)
Comment (mark-wiemer) [reply content/engine-dev-process/meetings.md:1111]:
```suggestion
    * `<sfan5>` probably me ([http://irc.minetest.net/minetest-dev/2018-03-31#i\_5264276](http://irc.minetest.net/minetest-dev/2018-03-31#i_5264276))
```

**Review by appgurueu — APPROVED** [review]

**Pull request merged**